node.py

```diff
@@ -1,8 +1,9 @@
 class Node:
-    def __init__(self, board, player, parent=None):
+    def __init__(self, board, player, parent=None, maximizingPlayer=False):
```

Comment:
Since you have your own node class now maybe remove the changes from this file too